sklearn/utils/__init__.py

```diff
@@ -56,6 +56,8 @@
 IS_PYPY = platform.python_implementation() == 'PyPy'
 _IS_32BIT = 8 * struct.calcsize("P") == 32
 
+MAX_INT_32 = np.iinfo(np.int32).max
```

Comment:
nitpick: `MAX_INT32` as `int32` is the datatype name in numpy.

Reply:
do we really need a constant for this?  `np.iinfo(np.int32).max` is short and explicit enough. But as you wish.

Reply:
Fine with me to be explicit but let's be explicit everywhere then